code/datums/martial/cqc.dm

```diff
@@ -19,7 +19,7 @@
 
 /datum/martial_art/cqc/can_use(mob/living/carbon/human/H)
 	var/area/A = get_area(H)
-	if(just_a_cook && !(istype(A, /area/crew_quarters/kitchen)))
+	if(just_a_cook && (!(istype(A, /area/crew_quarters/kitchen)) || !(istype(A, /area/crew_quarters/kitchen/coldroom))))
```

Comment:
if it's not a type of `/area/crew_quarters/kitchen`, or a type of `/area/crew_quarters/kitchen/coldroom`?

Reply:
Ah, didn't know that istype includes subtypes as well.